go/cmake/golang.cmake

```diff
@@ -10,7 +10,8 @@ function(GO_LIBRARY NAME BUILD_TYPE)
   else()
     set(BUILD_MODE -buildmode=c-shared)
     if(APPLE)
-      set(LIB_NAME "lib${NAME}.dylib")
+#      set(LIB_NAME "lib${NAME}.dylib")
+      set(LIB_NAME "lib${NAME}.so")
```

Comment:
Done, this is not a problem, we use _swig_paddle.so now, but I will not change this place in this pr, we can discuss it later.